src/main/webapp/src/app/create-gardens-form/create-gardens.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
I think we should put an Apache license header on this file -- one that is identical to the others in this repo.

Reply:
Done